demo/iaa-qpl-demo/Dockerfile

```diff
@@ -27,25 +26,24 @@ RUN curl -fsSL "$ACCEL_CONFIG_DOWNLOAD_URL" -o accel-config.tar.gz && echo "$ACC
 RUN cd idxd-config-accel-config-v$ACCEL_CONFIG_VERSION && \
     ./git-version-gen && \
     autoreconf -i && \
-    ./configure -q --libdir=/usr/lib64 --disable-test --disable-docs && \
+    ./configure -q --disable-test --disable-docs && \
     make install
 
-RUN git clone --recursive --depth 1 --branch v1.0.0 \
-    https://github.com/intel/qpl.git && \
+RUN git clone --recursive --depth 1 --branch v1.0.0 https://github.com/intel/qpl.git && \
     mkdir qpl/build && cd qpl/build && \
     sed -i '10i #include <stdexcept>' ../tools/benchmarks/include/types.hpp && \
     cmake -DLOG_HW_INIT=ON .. && \
-    cmake --build . --target install
+    make install
```

Comment:
QPL build instructions suggested to use `cmake --build ...` so I moved to use it